docs/operations_tasks/Ingest-Release-SOP.md

```diff
@@ -8,73 +8,79 @@ parent: Operations tasks
-3. Deploy tagged master branch
-4. Check for failed end-to-end tests
-5. Wrangler acceptance testing
-6. For production release, add version tag dYYYY-MM-DD.<release-count> (e.g. d2020-03-08.1).
-    - If there were 2 releases for that day, the second tag will be d2020-03-08.2
-7. Send message to AIT channel #hca when
-    - release is starting
-    - release is completed + release notes
-
-#### Release schedule
-- Staging Release : 10am every tues before sprint demo
-- Prod Release : 10am every tues mid sprint
-
-**Adhoc releases can happen and must be agreed upon by the team.**
-
-### Releasing hotfixes
-1. Inform the team of the hotfix. Team will decide if hotfix is needed and safe.
-2. Branch from the commit hash currently deployed in staging.
-3. Tag the commit. Deploy to staging. All tests must passed (DCP tests in staging, ingest tests, unit tests)
-5. Make a hotfix release notes
-7. Deploy to prod. Make sure all tests passed after hotfix release.
+
+### Pushing changes from dev, through staging, into production
+1. Create feature branch based from `dev`
+2. Commit your changes
+3. Create a pull request (PR) to `dev`
+4. Ensure unit tests and build steps are passing in [GitLab](https://gitlab.ebi.ac.uk/hca) for the project you're working on
+    - These will be reported in the PR UI in Github
+5. Get approvals from two other devs on the PR
+6. Merge PR
+7. Ensure deploy and integration tests in `dev` environment are passed in GitLab for the project
+8. Merge `dev` to `master`
+    - `git checkout dev && git pull && git checkout master && git pull && git merge dev && git push`
+9. Ensure deploy and integration tests in `staging` environment are passed in GitLab for the project
+10. Check deployment in [staging](https://staging.contribute.data.humancellatlas.org/) and consult with wranglers for feedback if neccesary
+11. Check with the team if it is okay to deploy this feature to production.
+12. Deploy what's on `staging` to `prod` via the operations panel in GitLab
+    1. Navigate to [GitLab](https://gitlab.ebi.ac.uk/) > `project your're working on` > operations > environments
+    2. Click "Deploy to" next to the `staging` environment
+    3. In the dropdown, select "Release prod"
+    4. Wait for the "Release prod" stage to complete (you can track the progress on the "CI/CD" page)
+    5. In the Environments page, select "Deploy to" again and then select the "Deploy prod" task in the dropdown
+    6. Track the deployment in the CI/CD page
+13. Ensure your changes are live on prod
+    - You may used the `kubectl get pods` command to check this
+    - Consult with [these docs](https://github.com/ebi-ait/ingest-kube-deployment) for further information on setting up `kubectl`
+
+### Manually deploying without GitLab
+You can manually deploy using the `ingest-kube-deployment` repo if GitLab is down and the deployment is urgent. 
```

Comment:
Let's make it clear that this is an emergency procedure. We should still make sure the PR is happening.